app.rb

```diff
@@ -0,0 +1,212 @@
+require_relative './person'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './classroom'
+
+# rubocop:disable Metrics
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def run
+    loop do
+      puts 'Welcome to School Library App!'
+      sleep 0.5
+      puts
+      puts 'Please choose an option by entering a number:'
+      puts '1 - List all books'
+      puts '2 - List all people'
+      puts '3 - Create a person'
+      puts '4 - Create a book'
+      puts '5 - Create a rental'
+      puts '6 - List all rentals for a given person id'
+      puts '7 - Exit'
+      puts
+
+      option = gets.chomp.to_i
+      user_option option
+    end
+  end
+
+  def user_option(input)
+    case input
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_a_person
+    when 4
+      create_a_book
+    when 5
+      create_a_rental
+    when 6
+      list_rentals
+    when 7
+      puts 'Thank you for using this app!'
+      exit
+    else
+      puts 'Invalid option, please enter a number between 1 and 7'
+    end
+  end
```

Comment:
- [ ] Great job so far managing the app and main files. However, the reason why there's a main.rb file is to handle the list of options presented to the user. Please refactor the app.rb file according to that.